package.tpl.xml

```diff
@@ -55,7 +55,7 @@ limitations under the License.
 	</notes>
 	<contents>
 		<dir name="/">
-			{% for file in files %}<file role="src" name="{{ file.path }}" />
```

Comment:
Why not just make role as an placeholder, instead of using filter?

Reply:
Use filters to improve code judgment.